### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by alexbchr**

### CODE REVIEW

**Comment by alexbchr** [comment]
@hansgx Updated code to use Identicon icons generated offline :wink: 

Works on both Web and Native.

File: src/components/images-and-icons/Avatar/Avatar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Avatar } from './Avatar'
+
+export default {
+  title: getStoryTitle(fileAbsolute),
+  component: Avatar,
+}
+
+export const Basic: React.FC = () => (
+  <Avatar
+    source={{ uri: 'https://cdn.pixabay.com/photo/2015/10/05/17/09/minion-972908_960_720.jpg' }}
```
Comment (hansgx) [inline src/components/images-and-icons/Avatar/Avatar.stories.tsx:13]: Pictures of Minions will not be tolerated in the codebase, please replace ASAP with [cute puppies](https://lmgtfy.com/?q=cute+puppy). 🐶

File: src/components/images-and-icons/Avatar/Avatar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  />
+)
+
+export const Generated: React.FC = () => <Avatar source={{ hash: 'John Doe' }} />
```
Comment (hansgx) [inline src/components/images-and-icons/Avatar/Avatar.stories.tsx:17]: Can you add two more?

File: src/components/images-and-icons/Avatar/Avatar.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+export const Small: React.FC = () => <Avatar size="small" source={{ hash: 'Julia Watson' }} />
```
Comment (hansgx) [inline src/components/images-and-icons/Avatar/Avatar.stories.tsx:18]: Maybe rename this Story to `Size` and show the different values?

File: src/components/images-and-icons/Avatar/Avatar.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import { Image, ImageSourcePropType } from 'react-native'
+import Identicon from 'identicon.js'
+import { Theme, useStyles, useTheme } from '../../../theme'
+import { generateSha512 } from '../../../utils/sha512'
+
+export interface GeneratedAvatarSource {
+  /**
+   * The image will be generated from this value using RoboHash.
+   *
+   * @see https://robohash.org/
```
Comment (hansgx) [inline src/components/images-and-icons/Avatar/Avatar.tsx:11]: Is RoboHash used behind the scenes, or is this a copy-paste error?

https://github.com/stewartlord/identicon.js

File: src/components/images-and-icons/Avatar/Avatar.stories.tsx (hunk 1 above)
Comment (alexbchr) [reply src/components/images-and-icons/Avatar/Avatar.stories.tsx:13]: Replaced with a cute puppy 

Cute puppy

File: src/components/images-and-icons/Avatar/Avatar.stories.tsx (hunk 2 above)
Comment (alexbchr) [reply src/components/images-and-icons/Avatar/Avatar.stories.tsx:17]: Done :wink:

File: src/components/images-and-icons/Avatar/Avatar.stories.tsx (hunk 3 above)
Comment (alexbchr) [reply src/components/images-and-icons/Avatar/Avatar.stories.tsx:18]: Done :wink:

File: src/components/images-and-icons/Avatar/Avatar.tsx (hunk 4 above)
Comment (alexbchr) [reply src/components/images-and-icons/Avatar/Avatar.tsx:11]: Nope, remnants from previous commits... :zombie_man: 

Replaced with Identicon.

**Pull request merged**